html-aam/index.html

```diff
@@ -16404,30 +16404,41 @@ <h4>Section and Grouping Element Accessible Name Computation</h4>
           </ol>
         </section>
         <section>
-          <h4>Text-level Element Accessible Name Computation</h4>
+          <h4>Name Prohibited Elements</h4>
+          <p>The following elements are prohibited from being named by authors.</p>
           <p>
-            <a>`abbr`</a>, <a>`b`</a>, <a>`bdi`</a>, <a>`bdo`</a>, <a>`br`</a>, <a>`cite`</a>, <a>`code`</a>, <a>`dfn`</a>, <a>`em`</a>, <a>`i`</a>, <a>`kbd`</a>, <a>`mark`</a>, <a>`q`</a>,
-            <a>`rp`</a>, <a>`rt`</a>, <a>`ruby`</a>, <a>`s`</a>, <a>`samp`</a>, <a>`small`</a>, <a>`strong`</a>,
-            <a data-cite="HTML/text-level-semantics.html#the-sub-and-sup-elements">`sub` and `sup`</a>, <a>`time`</a>, <a>`u`</a>, <a>`var`</a>, <a>`wbr`</a>
+            <a>`a`</a> <span class="el-context">(no `href` attribute)</span>, <a>`abbr`</a>, <a>`area`</a> <span class="el-context">(no `href` attribute)</span>,
+            <a data-cite="HTML">autonomous custom element</a> <span class="el-context">(if implicit `generic` role)</span>, <a>`b`</a>, <a>`bdi`</a>, <a>`bdo`</a>, <a>`br`</a>, <a>`body`</a>,
```

Comment:
removed br.  it was here by mistake and is already listed in the elements which cannot be named section.

```suggestion
            <a data-cite="HTML">autonomous custom element</a> <span class="el-context">(if implicit `generic` role)</span>, <a>`b`</a>, <a>`bdi`</a>, <a>`bdo`</a>,  <a>`body`</a>,
```